crates/bevy_scene/src/serde.rs

```diff
@@ -693,6 +693,31 @@ mod tests {
         assert_eq!(1, dst_world.query::<&Baz>().iter(&dst_world).count());
     }
 
+    #[test]
+    fn should_reserialize() {
```

Comment:
Can you add a link to the issue that this is a regression test for in a comment?